src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -94,7 +94,7 @@ export function AdvancedSwapDetails({ trade }: AdvancedSwapDetailsProps) {
               <SectionBreak />
               <AutoColumn style={{ padding: '0 24px' }}>
                 <RowFixed>
-                  <Text fontSize="14px">Route</Text>
+                  <Text fontSize='14px'>{TranslateString(232, 'Route')}</Text>
```

Comment:
This formatting is off. Should be double quotes.

Reply:
Done